docs/ref/models/querysets.txt

```diff
@@ -2703,6 +2703,15 @@ When :setting:`USE_TZ` is ``True``, datetime fields are converted to the
 current time zone before filtering. This requires :ref:`time zone definitions
 in the database <database-time-zone-definitions>`.
 
+.. fieldlookup:: week
+
+``week``
```

Comment:
This should probably also have the version annotation:

    .. versionadded:: 1.11